sonarqube_api/api.py

```diff
@@ -48,15 +49,22 @@ class SonarAPIHandler(object):
         'uncovered_conditions', 'coverage'
     )
 
-    def __init__(self, host=None, port=None, user=None, password=None):
+    def __init__(self, host=None, port=None, user=None, password=None,
+                 subdirectory=None, token=None):
```

Comment:
Could we rename `subdirectory` to `base_path` or something else that's less *filesystem*-ish?